app/views/devise/passwords/new.html.haml

```diff
@@ -5,7 +5,7 @@
 .govuk-grid-row
   .govuk-grid-column-one-third
     = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
-      = render "devise/shared/error_messages", resource: resource
+      = f.govuk_error_summary
```

Comment:
I'm not entirely sure why this change was made. It may just need adding to the commit message.

Reply:
Rewrote two commit messages, if you'd like to re-re-review please?